tutorials/nservicebus-101/nsb101-intro-paragraph.include.md

```diff
@@ -0,0 +1,3 @@
+The best way to get started with NServiceBus is to use it to build something realistic. In doing so you'll learn the architectural concepts behind the software, and start to learn its capabilities. In this step-by-step guide, you'll be building a back end for a retail e-commerce system. You'll learn how to send asynchronous messages between processes, how to use the Publish/Subscribe pattern to decouple business processes, and the advantages of using reliable messaging to enable automatic retries after processing failures.
```

Comment:
On guide vs. tutorial, https://github.com/Particular/DeveloperEducation/issues/775 will define branding and I'd prefer to not get hung up on it now. The word tutorial has not been used in any of the actual content thus far. The word "guide" is [defined as](http://www.dictionary.com/browse/guide?s=t) "a book, pamphlet, etc., giving information, instructions, or advice; handbook" so basically, insofar as it refers to "this thing you're reading" I definitely think it's fine for now.

On backend, I disagree. There is [some disagreement](https://dzone.com/articles/blogging-programmers-style) but the verdict seems to be to use "back end" as a noun and "back-end" as a compound adjective, but never "backend".

For the other text you mentioned, this include is like the second paragraph of a conference presentation abstract, so it has to tease and sound neat without the user necessarily knowing what all those things are.